FeedAPIChallenge/Feed API/FeedImageList.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  FeedImageList.swift
+//  FeedAPIChallenge
+//
+//  Created by Ajoy Kumar on 14/06/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal final class FeedImageList {
```

Comment:
The convention in this codebase is to omit the default access level keyword `internal`, so we can remove it.

Also, this class is an implementation detail of the `RemoteFeedLoader` when dealing specifically with JSON, but its current name doesn't relate to `remote` or its function `map`. It can be confusing to other developers since currently, it reads like a collection (`List`) of domain models (`FeedImage`), while it actually defines and works with remote models. I suggest renaming it to clarify its intent.
```suggestion
final class FeedImageList {
```